### PR TITLE
Added two listings, handwired keyboard and 3D printed case.

### PR DESCRIPTION
**Pull request by FilipParyz**
Added Freoduo handwired keyboard and case for Alpha PCB.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -118,6 +118,7 @@
 | [Atreis](https://github.com/dekonnection/atreis) | case | handwiring | <img src="https://github.com/dekonnection/atreis/raw/master/pictures/GPI0El4.jpg" width="200px" /> |
 | [Atreus Split Keyboard](https://www.thingiverse.com/thing:1646718) | case | 3d print, split, pro micro | <img src="https://cdn.thingiverse.com/renders/2a/35/f6/bd/51/ee77cfa99ac6c721974f6eb92a52cce9_preview_featured.jpg" width="200px" /> |
 | [Ergo-Warped](https://github.com/pseudoku/ErgoWarp) | case | 3d print, split, ergo | <img src="https://raw.githubusercontent.com/pseudoku/Warped-keyboard/master/Photo/Proto6.jpg" width="200px" /> |
+| [Freoduo](https://www.thingiverse.com/thing:4565609) | case | 3d print, split, ergo | <img src="https://i.imgur.com/Fvgm3Y5.jpg" width="200px" /> |
```
Comment (BenRoe) [inline docs/README.md:121]: Please move this one row down, so it's alphabetical sorted.

File: docs/README.md [hunk 2]
```diff
@@ -166,6 +167,7 @@
 | Name | Tags | Image |
 | --- | :---: | :---: |
 | [60% (Poker II, Pok3r) Keyboard Case 2.0](https://www.thingiverse.com/thing:983332) | 3d print | <img src="https://cdn.thingiverse.com/renders/8a/ac/f9/d8/ad/zanmYsB_-_Imgur_preview_featured.jpg" width="200px" /> |
+| [Alpha by Pyrol](https://www.myminifactory.com/object/3d-print-alpha-keyboard-case-pyrool-28-key-pcb-103767) | 3d print | <img src="https://cdn.myminifactory.com/assets/object-assets/5db6033fc6df3/images/720X720-cases.jpg" width="200px" /> |
```
Comment (BenRoe) [inline docs/README.md:170]: Please change the title to "Alpha keyboard case".

**Review by BenRoe — CHANGES_REQUESTED** [review]

**Comment by FilipParyz** [comment]
Hey @BenRoe I've changed what you requested :)

**Review by BenRoe — APPROVED** [review]
Looks good.
Thanks for your contribution.

**Pull request merged**